src/simple_calibration.jl

```diff
@@ -81,7 +83,60 @@ function simple_calibration_th228(e_uncal::Vector{<:Real}, th228_lines::Vector{<
     return result, report
 end
 
-
+"""
+    simple_calibration_gamma(e_uncal::Array, gamma_lines::Array, window_size::Float64=25.0, n_bins::Int=15000, quantile_perc::Float64=NaN, binning_peak_window::Float64=10.0, peak_quantile::ClosedInterval{<:Real} = 0.5..1.0)
+Perform a simple calibration for the uncalibrated energy array `e_uncal`
+* Find peaks within peak_quantile::ClosedInterval{<:Real} = 0.5..1.0 of the data
+* Estimate the calibration constant `c` by dividing the maximum gamma line energy by the peak guess
+* Return peakstats and peakhists of gamma lines for further calibration 
+"""
+function simple_calibration_gamma(e_uncal::Vector{<:Real}, gamma_lines::Vector{<:Unitful.Energy{<:Real}}, window_sizes::Vector{<:Tuple{Unitful.Energy{<:Real}, Unitful.Energy{<:Real}}},; n_bins::Int=15000, quantile_perc::Float64=NaN, binning_peak_window::Unitful.Energy{<:Real}=10.0u"keV", peak_quantile::ClosedInterval{<:Real} = 0.5..1.0)
+    e_min = quantile(e_uncal, leftendpoint(peak_quantile))
+    e_max = quantile(e_uncal, rightendpoint(peak_quantile))
+    # initial binning
+    bin_width = get_friedman_diaconis_bin_width(filter(in(e_min..e_max), e_uncal))
+    # create initial peak search histogram
+    h_uncal = fit(Histogram, e_uncal, 0:bin_width:maximum(e_uncal))
+    peak_guess, peak_idx = if isnan(quantile_perc)
+        h_peaksearch = fit(Histogram, e_uncal, e_min:bin_width:e_max)
+        # search all possible peak candidates
+        h_decon, peakpos = RadiationSpectra.peakfinder(h_peaksearch, σ=2.0, backgroundRemove=true, threshold=10)
+        # find the most prominent peak in the deconvoluted histogram
+        peakpos_idxs = StatsBase.binindex.(Ref(h_decon), peakpos)
+        cts_peakpos = h_decon.weights[peakpos_idxs]
+        peakpos[argmax(cts_peakpos)], argmax(cts_peakpos)
+    else
+        quantile(e_uncal, quantile_perc), length(gamma_lines)
+    end
+    @info "Identified most prominent peak (peak $(peak_idx), $(gamma_lines[peak_idx])). Peak guess: $peak_guess"
+    # get calibration constant for simple calibration
+    c = gamma_lines[peak_idx] / peak_guess
+    e_simple = e_uncal .* c
+    e_unit = u"keV"
+    # get peakhists and peakstats
+    peakhists, peakstats, h_calsimple, bin_widths = get_peakhists_th228(e_simple, gamma_lines, window_sizes; binning_peak_window=binning_peak_window, e_unit=e_unit)
```

Comment:
`get_peakhists_th228` also works for generic `simple_calibration_gamma`?
Or would that also need a `get_peakhists_gamma`?

Reply:
I wrote and tested this for my local Co60 data, but it should work for any other calibration spectra, because the anticipated gamma energies are passed as an argument. 
In principle we could consider re-writing/re-naming all `..._th228` calibration functions to be more generic for other gamma spectra. Some don't even need any modification at all and work out of the box for other sources. 
I didn't do that to keep the changes as minimal as possible.

Reply:
I agree. I can take care of this beginning of next year. 

In that context, I'd suggest to change the julia energy-calibration metadata as well: 
1) Add a field "source" in the metadata, which is "th228" for LEGEND-200, but can also be something else. Like that the dataflow can pick the correct `gamma_lines` and `gamma_names` from the energy calibration config. 

OR

2) Rename "th228_lines" etc in the metadata to "gamma_lines". 

I prefer option 1, because this allows you to switch easily back and fourth between different calibration sources.